src/Compilers/CSharp/Portable/Symbols/MissingCorLibrarySymbol.cs

```diff
@@ -22,13 +22,30 @@ internal sealed class MissingCorLibrarySymbol : MissingAssemblySymbol
     {
         internal static readonly MissingCorLibrarySymbol Instance = new MissingCorLibrarySymbol();
 
+        internal override TypeConversions TypeConversions
+        {
+            get
+            {
+                Debug.Assert(this == CorLibrary);
+
+                if (_lazyTypeConversions is null)
+                {
+                    Interlocked.CompareExchange(ref _lazyTypeConversions, new TypeConversions(this), null);
+                }
+
+                return _lazyTypeConversions;
+            }
+        }
+
```

Comment:
Please place this property below the constructor. We prefer keeping field declarations at the top of a type declaration. Also, constructors, usually, follows field declarations